image/src/app/config/application.py

```diff
@@ -1,12 +1,11 @@
 """
 All constants specific to the application
 """
-import serial
 from app.utils.env import env
 APPLICATION = {
     "PORT":      env("PORT","/dev/ttyUSB0"),
     "BAUD_RATE": env("BAUD_RATE","115200"),
-    "DATA_BITS": env("DATA_BITS",serial.EIGHTBITS),
-    "PARITY":    env("PARITY",serial.PARITY_NONE),
-    "STOP_BITS": env("STOP_BITS",serial.STOPBITS_ONE)
+    "DATA_BITS": env("DATA_BITS",8),
```

Comment:
why not use a named constant here? that never hurts

Reply:
because using the switch_demo which translates the variable environment from application.py

Reply:
the comment is related to line `8`, so `DATA_BITS`